openquake/qa_tests_data/classical_tiling/case_2/models/site/site_model.xml

```diff
@@ -8,7 +8,7 @@ xmlns:gml="http://www.opengis.net/gml"
         <site lat="37.54648" lon="34.98430" vs30="760.00" vs30Type="inferred" z1pt0="100" z2pt5="5"/>
         <site lat="36.91276" lon="35.17898" vs30="203.00" vs30Type="inferred" z1pt0="100" z2pt5="5"/>
         <site lat="36.97022" lon="35.42197" vs30="289.12" vs30Type="inferred" z1pt0="100" z2pt5="5"/>
-        <site lat="37.56571" lon="38.49118" vs30="457.76" vs30Type="inferred" z1pt0="100" z2pt5="5"/>
+        <site lat="37.56571" lon="38.49118" depth="-1.0000" vs30="457.76" vs30Type="inferred" z1pt0="100" z2pt5="5"/>
```

Comment:
Since we added a site with depth to this test, shouldn't there be depths in classical_tiling/case_2/expected/ hazard_map-mean.csv ? I only see lon and lat